plutus-core/plutus-ir/test/transform/computeArityTest/absapp.golden

```diff
@@ -1 +1 @@
-[]
\ No newline at end of file
+([], { (abs a (type) (lam x a x)) (con integer) })
```

Comment:
I'm unsure why this test is interesting - did you mean to test something different?

Reply:
Oh I see, it's just a copy of our set of random examples. I'd really prefer to have some specific interesting tests for this function, these don't seem that useful.

Reply:
Yeah this was to show type applications won't be counted. Maybe a bit overkill.